airbyte/_util/name_normalizers.py

```diff
@@ -68,6 +69,10 @@ def normalize(name: str) -> str:
         - "-1" -> "_1"
         """
         result = name
+        if not result:
```

Comment:
@aaronsteers this should probably log out a WARNING, but I dont immediately see the pattern for doing so in this repo.

Reply:
I think `warn_once` may be a good fit for this.

https://github.com/airbytehq/PyAirbyte/blob/679498794a1619906146c5d31390be9ef02f3015/airbyte/logs.py#L48-L58